examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,171 @@
+"""Fitting ODNP Datasets for Ksigma
+=================================== 
+The T1(p) and E(p) integrals are generated
+using the example, `generate_integrals.py`
+(as of 11/8/23 the `generate_integrals` is not
+updated), and stored in an H5 file.
+The inverse of the :math:`T_1(p)` data processing is taken
+to produce the :math:`R_1(p)` data which is fit using the
+polyfit of the inverse of krho with two degrees of
+freedom (*e.g.*, :math:`R_{100}(p) + \frac{C}{k_{\rho}^{-1}}`. 
+The :math:`R_1(p)` fit is then fed into the
+fitting routine for the integrals as a function of power
+prior to normalization. The cross relaxivity is then
+calculated using the output of these fits as well
+as the sample parameters that are fed to in (*e.g.*,
+ppt value, and concentration).
+"""
+from pyspecdata import *
+from sympy import symbols, Symbol, latex, lambdify
+from scipy.io import loadmat
+
+# {{{ This block changes when the data changes -- everything else should be
+# left alone for most circumstances
+data_info = dict(
+    filename="ras.h5",  # h5 file containing table of integrals for different datasets
+    data_dir="AG_processed_data",  # directory of the dataset of table of integrals
+    nodename="230706_M67_a",# specific nodename of the dataset of interest
+)  
+T100_info = dict(
+    filename="T10_DI_water_230412",
+    data_dir="AG_processed_data",
+)
+# }}}
+
+# {{{ load data
+integral_vs_p = find_file(
+    data_info["filename"],
+    exp_type=data_info["data_dir"],
+    expno=f"{data_info['nodename']}/Ep",
+)
+# {{{ Some older h5 files save the T1p rather than the R1p. If there isn't an
+# R1p expno then it will load the T1p integrals and convert to R1p by taking
+# the inverse
+try:
+    R1p = find_file(
+        data_info["filename"],
+        exp_type=data_info["data_dir"],
+        expno=f"{data_info['nodename']}/R1p",
+    )
+except:
+    T1p = find_file(
+        data_info["filename"],
+        exp_type=data_info["data_dir"],
+        expno=f"{data_info['nodename']}/T1p",
+    )
+    R1p = 1 / T1p
+# }}}
+# }}}
+# {{{ The powers go up, and then go back down in order to check for
+# reproducibility. Figure out where this flip occurs
+flip_idx = np.where(np.diff(integral_vs_p.fromaxis("power").data) < 0)[0][0] + 1
```

Comment:
`fromaxis("power").data` is equivalent to `getaxis("power")` here, so use the latter